src/test/java/io/appium/java_client/pagefactory_tests/MobileBrowserCompatibilityTest.java

```diff
@@ -44,7 +44,7 @@
     private AppiumDriverLocalService service;
 
     @AndroidFindBy(className = "someClass") @AndroidFindBy(xpath = "//someTag")
-    private RemoteWebElement btnG; //this element should be found by id = 'btnG' or name = 'btnG'
+    private RemoteWebElement btnK; //this element should be found by id = 'btnG' or name = 'btnG'
```

Comment:
If this is changed, then the comment should also be changed?

Reply:
ok. I will improve it a bit later.